teuthology/task/install/rpm.py

```diff
@@ -173,6 +173,13 @@ def _update_package_list_and_install(ctx, remote, rpm, config):
     :param rpm: list of packages names to install
     :param config: the config dict
     """
+
+    enable_coprs = config.get('enable_coprs', [])
+    if len(enable_coprs):
```

Comment:
This could also simply be `if enable_coprs:` since an empty list is falsey - no need to change this, of course.

Reply:
good to know, thanks